reference/orders.v3.yml

```diff
@@ -2784,79 +2718,22 @@ components:
         meta:
           $ref: '#/components/schemas/CollectionMeta'
       x-internal: false
-      x-examples:
-        example-1:
-          data:
-            - permission_set: app_only
-              namespace: 'minLength: 1'
-              key: 'minLength: 1'
-              value: 'minLength: 1'
-              description: 'minLength: 0'
-              resource_type: order
-              resource_id: 'minimum: 0'
-              id: 0
-              created_at: '2019-08-24T14:15:22Z'
-              updated_at: '2019-08-24T14:15:22Z'
-          meta:
-            pagination:
-              total: 36
-              count: 36
-              per_page: 50
-              current_page: 1
-              total_pages: 1
-              links:
-                previous: string
-                current: '?page=1&limit=50'
-                next: string
     MetafieldResponse:
       description: |
         Response payload for the BigCommerce API.
-      type: object
-      properties:
-        data:
-          $ref: '#/components/schemas/Metafield'
-        meta:
-          $ref: '#/components/schemas/Meta'
       x-internal: false
-      x-examples:
-        example-1:
-          data:
-            permission_set: app_only
-            namespace: 'minLength: 1'
-            key: 'minLength: 1'
-            value: 'minLength: 1'
-            description: 'minLength: 0'
-            resource_type: order
-            resource_id: 'minimum: 0'
-            id: 0
-            created_at: '2019-08-24T14:15:22Z'
-            updated_at: '2019-08-24T14:15:22Z'
-          meta:
-            meta:
-              total: 36
-              count: 36
-              per_page: 50
-              current_page: 1
-              total_pages: 1
-              links:
-                previous: string
-                current: '?page=1&limit=50'
-                next: string
+      x-examples: {}
+      allOf:
+        - type: object
+          properties:
+            data:
+              $ref: '#/components/schemas/Metafield'
+        - $ref: '#/components/schemas/Meta'
```

Comment:
Changed how the Meta model is referenced because it was nesting 'meta' within 'meta':

![image](https://user-images.githubusercontent.com/29524703/174170215-b0fe9248-e63e-4c71-bca4-7247c9ffd361.png)